jep/211/README.adoc

```diff
@@ -37,7 +37,8 @@ endif::[]
 
 | JIRA
 |
-link:https://issues.jenkins-ci.org/browse/JENKINS-52012[JENKINS-52012] - GA in weekly releases,
+link:https://issues.jenkins-ci.org/browse/JENKINS-52012[JENKINS-52012] - Preview in weekly releases,
+link:https://issues.jenkins-ci.org/browse/JENKINS-51805[JENKINS-52012] - GA in weekly releases,
```

Comment:
Typo in second link for GA: title is wrong.
@oleg-nenashev for next time: are you OK with me directly fixing such typos in your branch, or would you rather fix things like these yourself?